lib/identity_doc_auth/acuant/acuant_client.rb

```diff
@@ -15,8 +15,10 @@ def initialize(**config_keywords)
         @config = Config.new(**config_keywords)
       end
 
-      def create_document
-        Requests::CreateDocumentRequest.new(config: config).fetch
+      def create_document(cropping_mode:)
+        request = Requests::CreateDocumentRequest.new(config: config)
+        request.cropping_mode = cropping_mode
+        request.fetch
```

Comment:
> can we make `cropping_mode` a constructor argument to the CreateDocumentRequest? I don't like the pattern of mutate-after-create if we can avoid it

We could. I started going down that path initially, but paused and went this direction because I wasn't sure if we'd be okay with overloading the constructor with an argument that only applies for the subclass (not for `IdentityDocAuth::Acuant::Request`). I'm fine to change it back if it's not something we're concerned about though.

Reply:
yeah seems fine to me to override the constructor in the subclass, especially since we're calling that constructor directly here

Reply:
> can we make `cropping_mode` a constructor argument to the CreateDocumentRequest? I don't like the pattern of mutate-after-create if we can avoid it

Updated in 137db99.